### ISSUE TITLE
[sqlite] Unable to create column with 'NOT NULL' and without default value

### ISSUE THREAD

**Issue by gibat** [cakephp/phinx#1839]
**Steps to reproduce:**
1. Create migration
```
        $this->table('tableA', ['id' => false, 'primary_key' => ['id']])
            ->addColumn('id', 'integer', ['identity' => true])
            ->addColumn('textColumn', 'string', ['null' => true])
            ->create();

        $this->table('tableB', ['id' => false, 'primary_key' => ['id']])
            ->addColumn('id', 'integer', ['identity' => true])
            ->addColumn('textColumn', 'string', ['null' => false]) // <-- pay attention
            ->create();
```
2. Apply migrations

**Actual table structure**:
```
CREATE TABLE `tableA` (`id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT, `textColumn` VARCHAR NULL);
CREATE TABLE `tableB` (`id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT, `textColumn` VARCHAR NULL);
```

**Expected table structure**:
```
CREATE TABLE `tableA` (`id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT, `textColumn` VARCHAR NULL);
CREATE TABLE `tableB` (`id` INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT, `textColumn` VARCHAR NOT NULL); // <-- pay attention
```

**Additional information**
Possible reason in this line:
```
$def .= (!$column->isIdentity() && ($column->isNull() || is_null($default))) ? ' NULL' : ' NOT NULL';
```

**Comment by charlienux** [comment]
Fixed #1446
Please verify you are using the latest version of Phinx.

**Comment by gibat** [comment]
Yeah, you're right. Thanks!
